container/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY include/ include/
 COPY LICENSE ./
 COPY README.md ./
 COPY inputfile/ inputfile/
-RUN cmake . && make
+RUN cmake . -DCMAKE_BUILD_TYPE=RELEASE && make
```

Comment:
I added this cmake flag because the default build will be the (very slow) debug build. Please make sure that this doesn't break anything because I have no means of testing.